deploy/overlays/private/memdb.json

```diff
@@ -0,0 +1,105 @@
+{
+  "importName": "Feeding America",
+  "provenanceUrl": "https://www.feedingamerica.org/",
+  "dataDownloadUrl": "https://www.feedingamerica.org/",
+  "rootSvg": "g/Feeding_America",
```

Comment:
This is fine for now.

(1) ... I think private DC people will probably need to deal with TMCF/MCF anyway?  But I see the point in avoiding code to build the proto-like thing :).  Perhaps we could perhaps simplify by not requiring separate "displayName" vs. "searchNames" etc?

(3) ... I just asked because this was very Feeding America specific, and only supported one import :)

Reply:
(1) We could definitely simply the config requirement, so it minimally just asks a list of DCID and names (even names could be omitted then DCID is used to display).